mmdet3d/models/middle_encoders/sparse_encoder.py

```diff
@@ -66,7 +67,10 @@ def __init__(self,
                 conv_type='SubMConv3d')
 
         encoder_out_channels = self.make_encoder_layers(
-            make_sparse_convmodule, norm_cfg, self.base_channels)
+            make_sparse_convmodule,
+            norm_cfg,
+            self.base_channels,
+            option=option)
```

Comment:
1. option -> block_type
2. update docstrings accordingly
3. assert the available options of `block_type`